docs/data/sdks/ios/index.md

```diff
@@ -427,6 +427,192 @@ If the user property doesn't exist, it's initialized to an empty list before the
+    `defaultTracking.screenViews` | Optional. `boolean` | `NO` | Enables screen views tracking. If value is `YES`, Amplitude tracks screen viewed events.<br /><br />Event properties tracked includes: `[Amplitude] Screen Name`<br /><br />See [Tracking screen views](#tracking-screen-views) for more information.|
+    `defaultTracking.deepLinks` | Optional. `boolean` | `NO` | Enables deep link tracking. If value is `YES`, Amplitude tracks deep link opened events. Note that you still need to call `continueUserActivity` or `openURL` manually for tracking this event.<br /><br />Event properties tracked includes: `[Amplitude] Link URL`, `[Amplitude] Link Referrer`<br /><br />See [Tracking deep links](#tracking-deep-links) for more information.|
+
+You can enable Amplitude to start tracking all events mentioned above, use the code sample below.
+
+=== "Objective-C"
+
+    ```obj-c
+    [Amplitude instance].defaultTracking = [AMPDefaultTrackingOptions initWithAllEnabled];
+    [[Amplitude instance] initializeApiKey:@"API_KEY"];
+    ```
+
+=== "Swift"
+
+    ```swift
+    Amplitude.instance().defaultTracking = AMPDefaultTrackingOptions.initWithAllEnabled()
+    Amplitude.instance().initializeApiKey("API_KEY")
+    ```
+
+!!!note
+    Amplitude may add more events in a future version, and this configuration enables tracking for those events as well.
+
+Similarly, you can disable Amplitude to track all events mentioned above with the code sample below.
+
+=== "Objective-C"
+
+    ```obj-c
+    [Amplitude instance].defaultTracking = [AMPDefaultTrackingOptions initWithNoneEnabled];
+    [[Amplitude instance] initializeApiKey:@"API_KEY"];
+    ```
+
+=== "Swift"
+
+    ```swift
+    Amplitude.instance().defaultTracking = AMPDefaultTrackingOptions.initWithNoneEnabled()
+    Amplitude.instance().initializeApiKey("API_KEY")
+    ```
+
+You can also customize the tracking with `DefaultTrackingOptions`, see code sample below.
+
+=== "Objective-C"
+
+    ```obj-c
+    [Amplitude instance].defaultTracking = [AMPDefaultTrackingOptions initWithSessions:YES
+                                                                         appLifecycles:NO
+                                                                             deepLinks:NO
+                                                                           screenViews:NO];
+    [[Amplitude instance] initializeApiKey:@"API_KEY"];
+    ```
+
+=== "Swift"
+
+    ```swift
+    Amplitude.instance().defaultTracking = AMPDefaultTrackingOptions.initWithSessions(true, appLifecycles: false, deepLinks: false, screenViews: false)
```

Comment:
I think it might help to add line breaks for the function args, right now you have to scroll a lot to see the relevant parts of the code.

<img width="823" alt="image" src="https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/5790872/e880e95e-1839-46bc-a2bf-43cd8419c703">

```suggestion
    Amplitude.instance().defaultTracking = AMPDefaultTrackingOptions.initWithSessions(
    true,
    appLifecycles: false,
    deepLinks: false,
    screenViews: false
 )
```